charactersheet/charactersheet/models/common/weapon.js

```diff
@@ -136,6 +136,27 @@ function Weapon() {
         }
     });
 
+    self.weaponRangeLabel = ko.pureComputed(function() {
+        if (self.weaponType().toLowerCase() === 'ranged'){
```

Comment:
Just in the sense by saying if 'ranged' then else, we would be letting anything that's not 'ranged' pass through, which could cause silent errors.  Even though for now it's only ranged or melee it would be an assumption.  I'm open to changing it if it's our convention, but I am of the opinion that it's okay the way it is.